src/auth/auth.controller.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Controller,
+  HttpStatus,
+  Get,
+  Query,
+  HttpException,
+  Post,
+  UseGuards,
+  Req,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { JwtAuthGuard } from './jwt-auth.guard';
+import { TRequest } from 'src/common/types/user.types';
```

Comment:
PrimathonAICodeReviewer

The import path for `TRequest` seems incorrect. It should be relative to the current file's location.
```diff
- import { TRequest } from 'src/common/types/user.types';
+ import { TRequest } from '../common/types/user.types';
```



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->